$OpenBSD: patch-src_libopensc_cardctl_h,v 1.1 2010/12/27 15:30:11 jasper Exp $

Provide a definition of MIN() in case it's not defined already.

--- src/libopensc/cardctl.h.orig	Mon Dec 27 13:11:32 2010
+++ src/libopensc/cardctl.h	Mon Dec 27 13:11:47 2010
@@ -740,6 +740,10 @@ typedef struct sc_rtecp_genkey_data {
 	unsigned char  *invq;
 };
 
+#ifndef MIN
+#define MIN(a,b) (((a) < (b)) ? (a) : (b))
+#endif
+
 #ifdef __cplusplus
 }
 #endif
